src/assets/AssignedCategoriesIcon.js

```diff
@@ -0,0 +1,67 @@
+import React from 'react'
+
+const AssignedCategoriesIcon = () => {
+    return (
+        <svg width="12px" height="12px" viewBox="0 0 12 12">
+            <title>assigned-categories</title>
```

Comment:
Minor thing.
I usually clean up a bit the SVG I get, removing title and ids and stuff not really needed.